# Fix makefile to use EB values
# Ward Poelmans <wpoely86@gmail.com>
--- SOAPdenovo-Trans-src-v1.04.orig/src/Makefile	2014-04-15 04:37:16.000000000 +0200
+++ SOAPdenovo-Trans-src-v1.04/src/Makefile	2017-08-23 07:54:58.108664416 +0200
@@ -1,5 +1,5 @@
-CC=             gcc
-CFLAGS=         -O3 -fomit-frame-pointer -static
+CC := $(CC)
+CFLAGS := -fomit-frame-pointer $(CFLAGS)
 #CFLAGS=         -O3 -g -D_DEBUG
 DFLAGS= 
 
@@ -32,7 +32,7 @@
 INCLUDES=	-Iinc
 SUBDIRS=        . 
 LIBPATH=	
-LIBS=      	-pthread -lm -lrt -lbam -lz -L./inc
+LIBS :=      	$(LIBS) -lrt -lbam -lz -L./inc
 EXTRA_FLAGS=
 
 BIT_ERR = 0
@@ -73,7 +73,7 @@
 		@test $(BIT_ERR) != 1 || sh -c 'echo "Fatal: 64bit CPU and Operating System required!";false;'
 
 SOAPdenovo:	envTest $(OBJS)
-		@$(CC) $(CFLAGS) -o $(PROG) $(OBJS) $(LIBPATH) $(LIBS) $(ENTRAFLAGS)
+		$(CC) $(CFLAGS) $(LDFLAGS) -o $(PROG) $(OBJS) $(LIBPATH) $(LIBS) $(ENTRAFLAGS)
 		@printf "Linking...\r"
 		@printf "$(PROG) compilation done.\n";
 
